src/qfs.ml

```diff
@@ -179,25 +179,79 @@ type system_info = {
   uptime : int;
   buffers : int;
   clients : int;
+  delayed_recovery : int;
+  in_recovery : int;
   servers : int;
+  sockets : int;
   chunks : int;
   requests : int;
   total_drives : int;
   writable_drives : int;
   max_clients : int;
   max_servers : int;
   buffers_total : int;
+  pending_recovery : int;
   pending_replication : int;
   replication_backlog : int;
+  repl_check_timeouts : int;
   replications : int;
   replications_check : int;
   fattr_nodes : int;
 }
 
 type server =
 {
-  host : string;
+  ip : string;
```

Comment:
this is better field name yes, but it breaks compatibility, better avoid this change